tracing-attributes/src/lib.rs

```diff
@@ -853,20 +864,67 @@ impl Parse for Level {
     }
 }
 
-fn param_names(pat: Pat) -> Box<dyn Iterator<Item = Ident>> {
+/// A enum indicates the field should be recorded as `Value` or `Debug`.
+enum RecordType {
+    /// Represents the field should be recorded as it is.
+    Value,
+    /// Represents the field should be recorded as `tracing::field::debug()`.
+    Debug,
+}
+
+impl RecordType {
+    /// Array of primitive types which should be recorded as [RecordType::Value].
+    const TYPES_FOR_VALUE: [&'static str; 11] = [
+        "bool", "u8", "i8", "u16", "i16", "u32", "i32", "u64", "i64", "usize", "isize",
```

Comment:
There are a few additional primitive value types we can handle here --- see [here](https://docs.rs/tracing/0.1.25/tracing/field/trait.Value.html#foreign-impls) for the list of types implementing the trait.

In particular, I think we can handle `NonZero{I8, I16, I32, I64, Isize, U8, U16, U32, U64, Usize}`, `Wrapping<T>` where `T` is a primitive type, and `&str` here as well.